api/apps/api/src/modules/projects/projects.controller.ts

```diff
@@ -323,13 +325,21 @@ export class ProjectsController {
   @Get(':id/calibration')
   async getProjectBlmValues(
     @Param('id') id: string,
+    @Req() req: RequestWithAuthenticatedUser,
   ): Promise<ProjectBlmValuesResponseDto> {
-    const result = await this.projectsService.findProjectBlm(id);
+    const result = await this.projectsService.findProjectBlm(id, req.user.id);
 
     if (isLeft(result))
-      throw new NotFoundException(
-        `Could not find project BLM values for project with ID: ${id}`,
-      );
+      switch (result.left) {
+        case projectNotFound:
+          throw new NotFoundException(
+            `Could not find project BLM values for project with ID: ${id}`,
+          );
+        case forbiddenError:
+          throw new ForbiddenException();
+        default:
+          throw new InternalServerErrorException();
+      }
```

Comment:
nothing to do here and now, but I think we'll end up with a lot of almost identical boilerplate in most controller handlers, eventually, as we phase in access control throughout - it may be a good idea to think about a utility function that handles at least the default cases we expect everywhere (or nearly) such as not found, forbidden and "all the rest"...